### PR TITLE
Circular import fixes, part 6: assorted imports and async reducer initial state

### PR DESCRIPTION
**Pull request by markerikson**
Part of #6673 for fixing circular imports.

This PR:

- Moves a couple of the "async initial state" functions out of their reducer files and into `ui/setup/index.ts`, which is the only file that uses them.  This fixes some circles due to importing prefs functions into reducers.
- Copy-pastes a couple trivial selectors
- Reworks the `visibleColumnBreakpoints` file and associated code
  - There was a `selectedSource` param being passed through several functions, but turns out it was never actually being used. Removed that param from all related functions
- General rearrangement of several other imports

Thanks to the `circular-webpack-plugin` output changes (which I haven't committed yet), this PR gets us down to "only" 49 cycles, and we started with 174.  So, 72% fixed so far.

### CODE REVIEW

File: src/devtools/client/debugger/src/reducers/tabs.ts [hunk 1]
```diff
@@ -31,13 +29,7 @@ export interface TabsState {
...
 }
 
-const EMPTY_TABS: Tab[] = [];
-
-export const getInitialTabsState = async () => {
-  const session = await getReplaySession(getRecordingId()!);
-
-  return { tabs: session?.tabs ?? EMPTY_TABS };
-};
+export const EMPTY_TABS: Tab[] = [];
```
Comment (jaril) [inline src/devtools/client/debugger/src/reducers/tabs.ts:32]: Why not move this along with getInitialTabsState?

Comment (jaril) [reply src/devtools/client/debugger/src/reducers/tabs.ts:32]: Wait I just saw it nevermind!

File: src/devtools/client/debugger/src/utils/editor/get-token-location.js [hunk 2]
```diff
@@ -3,16 +3,3 @@
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
 //
```
Comment (jaril) [inline src/devtools/client/debugger/src/utils/editor/get-token-location.js:5]: Can we delete this file then?

**Review by jaril — APPROVED** [review]

File: src/devtools/client/debugger/src/utils/editor/get-token-location.js (hunk 2 above)
Comment (markerikson) [reply src/devtools/client/debugger/src/utils/editor/get-token-location.js:5]: Oh yeah, I should have done that earlier! :)

**Pull request merged**